app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({self.name: self})
```

Comment:
Ensure that the names used as keys in `Person.people` are unique to prevent overwriting existing entries. You might want to check if the name already exists in the dictionary before updating it.